src/ticketmap/index.tsx

```diff
@@ -369,54 +391,48 @@ export default class TicketMap extends Component<Props, State> {
     this.setState({ selectedSections })
   }
 
-  highlightZone = (zone: string) => {
-    return this.toggleZoneHighlight(zone, true)
-  }
+  highlightZone = (zone: string): void => this.toggleZoneHighlight(zone, true)
 
-  unhighlightZone = (zone: string) => {
-    return this.toggleZoneHighlight(zone, false)
-  }
+  unhighlightZone = (zone: string): void => this.toggleZoneHighlight(zone, false)
 
-  toggleZoneHighlight = (zone: string, shouldHighlight = true) => {
+  toggleZoneHighlight = (zone: string, shouldHighlight: boolean = true): void => {
```

Comment:
The default value of `shouldHighlight` is already hinting at its type, so the explicit hint just adds noise.

```suggestion
  toggleZoneHighlight = (zone: string, shouldHighlight = true): void => {
```